restapi/src/main/java/io/stargate/web/docsapi/dao/DocumentDB.java

```diff
@@ -314,9 +255,13 @@ public void tableExists(String keyspaceName, String tableName) {
     }
   }
 
+  private List<String> allColumnNames() {
+    return Arrays.asList(DocsApiConstants.ALL_COLUMNS_NAMES.apply(config.getMaxDepth()));
```

Comment:
I was thinking that such things we can extract into immutable class as I specified in the #1235. Since, this is too big maybe we can keep that issue open and just do that on top of this PR. Wdyt?